lib/rust/json-rpc/src/log.rs

```diff
@@ -0,0 +1,33 @@
+//! Interface for logging RPC Requests as [`enso_profiler`] metadata, and interpreting the resultant
+//! log events.
+
+
+
+// ===========================
+// === Rpc Request Logging ===
+// ===========================
+
+/// Log an RPC Request to the profiling framework.
+pub fn rpc_request(method: &'static str) {
```

Comment:
I think we start using this pattern quite a lot. I wonder if the boilerplate can be reduced but abstracting this to a method that creates a function for logging metadata of a certain type, or a macro.